pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: watcher
-version: 0.9.8-dev
+version: 0.9.7+1
```

Comment:
This would have been 0.9.7+1-dev if such a version worked as expected. Since it doesn't, we add `-dev` to the next lowest version that *is* compatible and move back to the build identifier when we actually cut a release.